config/schema/rules.schema.yml

```diff
@@ -219,15 +219,15 @@ rules.settings:
   type: config_object
   label: 'Rules settings'
   mapping:
-    log_errors:
-      type: integer
-      label: 'Logging of Rules evaluation errors'
-    debug_log:
+    log:
       type: boolean
-      label: 'Log debug information to the available loggers'
-    debug:
+      label: 'Log debug information'
+    log_level_system:
+      type: string
+      label: 'Log level'
+    debug_screen:
       type: boolean
-      label: 'Show debug information'
-    log_level:
-      type: integer
+      label: 'Log debug information'
```

Comment:
Wrong label. This should be "Show debug information on screen (in the HTML response)"